app/src/main/java/com/example/checkpoint/dto/Route.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.checkpoint.dto
+
+import android.text.method.DateTimeKeyListener
+
+open class Route(
+    userId: Int,
+    name: String,
+    longitude: String,
+    latitude: Int,
+    userLocation: String,
+    routeTypeId: Int,
+    destination: String
+) : Map(userId, name, longitude, latitude, userLocation)
+
+class Event:Route{
+    constructor(userId: Int,name: String,longitude: String,latitude: Int,userLocation: String,routeTypeId: Int,destination: String,eventId:Int,Location:String,Time:DateTimeKeyListener):
+            super(userId,name,longitude,latitude,userLocation,routeTypeId,destination)
+}
```

Comment:
Because the lack of dto in the program can affect the normal use of data, a dto environment is created for use according to the architecture of UML diagram.